service.py

```diff
@@ -683,9 +685,10 @@ def error(message):
 
 @app.route('/author/<author_id>')
 def author(author_id):
+    author = Author.query.filter_by(id=author_id).first()
```

Comment:
Is this line for adding an author column to the list of articles?

Thanks.

Reply:
@hema-001 
This one is for that 'Salman Khan' fix, here we just retrieve the `Author` with the id `author_id` and then pass the result on to the author template.